docs/source/cli_help/advance.rst

```diff
@@ -0,0 +1,65 @@
+.. role:: bash(code)
+
+.. _advance:
+
+advance
+-------
+Usage:
+
+.. code-block:: shell
+
+    git machete advance [-y|--yes]
+
+Fast forwards (as in :bash:`git merge --ff-only`) the current branch :bash:`C` to match its downstream :bash:`D`,
+and subsequently slides out :bash:`D`. Both steps require manual confirmation unless :bash:`-y/--yes` is provided.
+
+The downstream :bash:`C` is selected according to the following criteria:
+    * if :bash:`C` has exactly one downstream branch :bash:`d` whose tip is a descendant of :bash:`C`, and whose fork point is equal to :bash:`C` or is overridden
```

Comment:
I'm re-reading this stuff of mine now... this point is super-nerd-level technical 🤓 I'd simplify it to just:
```suggestion
    * if :bash:`C` has exactly one downstream (child) branch :bash:`d` connected with a green edge to `C`, then...
```

Reply:
Maybe `green edge` could link to help for `status` where the exact definition of green edge is provided tho (?)